src/components/explore/GenerationResults.tsx

```diff
@@ -169,7 +172,7 @@ const GenerationResults: React.FC<any> = observer((props) => {
                             <div className="flip-card-back items-center h-screen">
                               {buttonHide(workflow)}
                               <h3>{ workflow.descriptive_name }</h3>
-                              <h4>Technical benchmarks</h4>
+                              <h4>Design-time benchmarks</h4>
```

Comment:
We might want to have a tooltip, providing a definition for "design-time benchmarks".